### DISCUSSION TITLE
Get SINGLE:NO_TRAFFIC

### DISCUSSION THREAD

**Discussion by mor-sound** [pfelk/pfelk#329] [Q&A]
Hey all!
the pfSense Firewall dosnt send syslog to pfelk. 
In pfTop i get SINGLE:NO_TRAFFIC, any suggestions why?

**Comment by a3ilson** [comment]
Based on what you provided, it doesn't appear you properly configure remote logging.

**Comment by mor-sound** [comment]
This is how I configed that:
checkbox "send log messages to remote syslog server"
In source Address- LAN
IP Protocol- IPv4
Remote log servers- <pfelk- Server IP>:5140
Chekbox- Evrithing

![Capture](https://user-images.githubusercontent.com/30343238/129992345-98fba112-3b5a-4d53-bfe7-dc8389d72d07.PNG)

Thanks for your help :)

**Reply by a3ilson** [reply]
On the destined remote log server are you receiving logs (e.g. tcpdump)? Is your pfSense on the same network as the remote log server or does it travers through a router?

**Comment by mor-sound** [comment]
the pfelk is in LAN network and its directly connected to pfsence,
I can see the trafic in my pfelk machine.

**Reply by a3ilson** [reply]
What version of pfSense are you running, how did you install pfelk, can you provided the log file?

**Comment by mor-sound** [comment]
My pfSense virtion 2.4.5 p1
I used script method to install pfELK

**Reply by mor-sound** [reply]
*version
